learning/src/components/WordDefinition.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from "react";
+import { render } from "react-dom";
+
+function SoundButton(props) {
```

Comment:
Since we are using react, there's a more js way of implementing this. See 952f6f97bacc8c239beac8357a4cf1c1e4a99364